components/quick_input.jsx

```diff
@@ -89,7 +93,9 @@ export default class QuickInput extends React.PureComponent {
             inputComponent || 'input',
             {
                 ...props,
-                ref: 'input',
+                ref: (input) => {
```

Comment:
I'm 0/5 on whether or not we use a string ref or a function one, but we should make this a setInput method instead of making the new arrow function each time